### PR TITLE
Update analytics code and add IE11 legacy scripts

### PR DESCRIPTION
**Pull request by tomodwyer**
- Target IE11 with legacy scripts
- Add analytics code to `main.js` file
- Move IE11 detection to `main.js` file

### CODE REVIEW

File: templates/base.html [hunk 1]
```diff
@@ -53,16 +50,12 @@
...
   {# Legacy browsers #}
-  <script nomodule>
-    var isIE11 = !!window.MSInputMethodContext && !!document.documentMode;
-    if (isIE11) { document.documentElement.classList.add('ie11'); }
-  </script>
   <script nomodule src="https://polyfill.io/v3/polyfill.min.js"></script>
   <script nomodule src="https://cdnjs.cloudflare.com/ajax/libs/systemjs/6.8.3/system.min.js" integrity="sha512-dGQpe9Hb1dXjYT7TSrTS6qnRmzrhY9wsyRtN6BR2R+XgiXf5cpgvWVGPWptrarkfoldzpIkl7adWEddhoKsX6A==" crossorigin="anonymous" referrerpolicy="no-referrer"></script>
   <script nomodule src="https://cdnjs.cloudflare.com/ajax/libs/alpinejs/2.8.2/alpine-ie11.js" integrity="sha512-KA2IvqHkHQMVbJlMnypXkjiHFkYAVaYSd193ckoVZYZ7AH2qZqlv7odiJcjAF2lECSEHptHFrrKaIWCXqZCXHg==" crossorigin="anonymous" referrerpolicy="no-referrer"></script>
 
+  {# {% vite_asset "vite/legacy-polyfills" scripts_attrs=scripts_attrs %} #}
```
Comment (ghickman) [inline templates/base.html:57]: Did you mean to leave this comment in?

**Review by ghickman — APPROVED** [review]
👍

File: templates/base.html (hunk 1 above)
Comment (tomodwyer) [reply templates/base.html:57]: Ah, good spot, I did not!

Comment (tomodwyer) [reply templates/base.html]: It was causing errors in IE11, but I've added the required polyfills in https://github.com/opensafely-core/output-explorer/pull/160/commits/2f831ecd3c706128b2d97ddd6a0f00474661423f to mitigate the errors.

**Pull request merged**